addons/binding/org.openhab.binding.hdpowerview/src/main/java/org/openhab/binding/hdpowerview/handler/HDPowerViewHubHandler.java

```diff
@@ -123,7 +123,7 @@ private void schedulePoll() {
             pollFuture.cancel(false);
         }
         logger.debug("Scheduling poll for 500ms out, then every {} ms", refreshInterval);
-        pollFuture = scheduler.scheduleAtFixedRate(pollingRunnable, 500, refreshInterval, TimeUnit.MILLISECONDS);
+        pollFuture = scheduler.scheduleWithFixedDelay(pollingRunnable, 500, refreshInterval, TimeUnit.MILLISECONDS);
```

Comment:
Please do not yet touch the MILLISECONDS in most cases there are some stricter timings needed in those. Can you check others as well?

Reply:
@martinvw What exactly do you mean by this? That anything that is scheduled more frequently than once a second should be done at fixed rate and not with fixed delay?

Reply:
Not completely, but everything that is scheduled so often might have a valid use case. In all others minute-resolution etc I'm sure that it does not matter.

Reply:
Note that in this specific case the default is set 60 000 so it does not matter, but there is for example some fading logic (I believe in the WIFI led binding) which might break by changing it to a fixedDelay.

Reply:
Well, especially these cases are risky as if the job takes too long, you might end up running multple in parallel - and when debugging and stepping through your code, the fixed rate is pretty nasty as well.
So unless there is some requirement that it has to be exactly 500ms, I would vote for changing it - and imho that should be ok for almost all places in the code.

Reply:
> some fading logic

Yes, you are right, that would be a valid example where we'd need to keep the fixedRate.

Reply:
@kaikreuzer  , @martinvw 

Could you please specify all the cases in which scheduleAtFixedRate should not be changed, or  tell me how to determine if I should not change them.

Additionally, I could add a white list of allowed use cases for scheduleAtFixedRate calls in the checkstyle rule.

Reply:
I marked the one I could find the rest should be safe to change imho